ext/date/php_date.h

```diff
@@ -129,7 +129,8 @@ ZEND_END_MODULE_GLOBALS(date)
 
 #define DATEG(v) ZEND_MODULE_GLOBALS_ACCESSOR(date, v)
 
-PHPAPI time_t php_time(void);
+/* DEPRECATED, use zend_realtime_get() instead */
+#define php_time() zend_realtime_get()
```

Comment:
Can you please leave this as a (inline) function and add `ZEND_ATTRIBUTE_DEPRECATED` so that folks actually notice that this is deprecated?